assets/src/edit-story/components/library/libraryUploadDropTarget.js

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Can we combine this with `WorkspaceUploadDropTarget` to become `WithUploadDropTarget` (or even shorter `WithUploadZone`) and handle whether the element is inserted or not after upload through props?

Reply:
The reason why these are split into sub-components is because the layout system is complicated. In case of canvas overlay, a panel is first display to hide the page's content (white), then the screen shown, and finally the upload message is overlayed perfectly centered over the page area. That's the reason why all three components are separated here.